core/src/epicli/data/common/validation/configuration/postgres-exporter.yml

```diff
@@ -0,0 +1,5 @@
+# /usr/local/epicli/data/common/validation/configuration/postgres-exporter.yml
```

Comment:
:)

1. Why is the path here for?
2. This does not look like an yaml representation of a json schema document :thinking: 
3. My claim from 2. may be incorrect, can you prove it does really work?
4. What do we want to achieve here? Do we want user to always enable exporter and always set the same port? :thinking:
5. (BONUS) Identation could be improved :thinking:

Reply:
I think we should forget about this content... No need to validate this: